lms/djangoapps/course_home_api/outline/views.py

```diff
@@ -275,14 +276,14 @@ def get(self, request, *args, **kwargs):  # pylint: disable=too-many-statements
             if allow_public or user_is_masquerading:
                 handouts_html = get_course_info_section(request, request.user, course, 'handouts')
 
-        if not show_enrolled:
+        if not is_enrolled:
             if CourseMode.is_masters_only(course_key):
                 enroll_alert['can_enroll'] = False
                 enroll_alert['extra_text'] = _(
                     'Please contact your degree administrator or '
                     '{platform_name} Support if you have questions.'
                 ).format(platform_name=settings.PLATFORM_NAME)
-            elif course.invitation_only:
+            elif course_is_invitation_only(course):
```

Comment:
This and the rest of the PR is a tiny cleanup. I noticed that some places were only checking `.invitation_only`, but others were additionally checking a setting toggle that forced all courses to be invitation-only.

So I added a utility method and made all current checks use it. No known bug (access checks might have made this accidentally work). But just better to be consistent in how we check it for copy-paste reasons.